<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VASP Input Selector</string>
  </property>

  <!-- Toggle checkbox for switching modes -->
  <widget class="QCheckBox" name="toggleModeCheckbox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Toggle Manual Build Mode</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>

  <!-- Pseudopotential selection label and list -->
  <widget class="QLabel" name="potcarLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Pseudopotentials</string>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="visible">
    <bool>false</bool> <!-- Hidden initially -->
   </property>
  </widget>

  <widget class="QListWidget" name="potentialList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>200</width>
     <height>250</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="visible">
    <bool>false</bool> <!-- Hidden initially -->
   </property>
  </widget>

  <widget class="QPushButton" name="finalizeSelectionButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Finalize Selection</string>
   </property>
   <property name="visible">
    <bool>false</bool> <!-- Hidden initially -->
   </property>
  </widget>

  <!-- File picker buttons -->
  <widget class="QPushButton" name="incarPicker">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>80</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Pick INCAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="poscarPicker">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>150</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Pick POSCAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="potcarPicker">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>220</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Pick POTCAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runCalculations">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>290</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Calculations</string>
   </property>
  </widget>

  <!-- Additional POSCAR input fields -->
  <widget class="QLabel" name="poscarLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>POSCAR Parameters</string>
   </property>
  </widget>
  
  <widget class="QLineEdit" name="latticeParameterInput">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Lattice Parameter</string>
   </property>
  </widget>

  <widget class="QLineEdit" name="atomCountInput">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Atom Count</string>
   </property>
  </widget>

  <widget class="QLineEdit" name="atomTypeInput">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Atom Type</string>
   </property>
  </widget>
</widget>
 <resources/>
 <connections/>
</ui>
